dial_opts.go

```diff
@@ -5,56 +5,108 @@ import (
 	"context"
 	"crypto/tls"
 	"crypto/x509"
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
 
 	"github.com/certifi/gocertifi"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 )
 
 const (
-	// SkipVerifyCA is a constant that imrpoves the readability of `WithSystemCerts`
+	// SkipVerifyCA is a constant that improves the readability of functions
+	// with the insecureSkipVerify parameter.
 	SkipVerifyCA = true
 
-	// VerifyCA is a constant that improves the readability of `WithSystemCerts`.
+	// VerifyCA is a constant that improves the readability of functions
+	// with the insecureSkipVerify parameter.
 	VerifyCA = false
 )
 
-// WithSystemCerts is a dial option for requiring TLS with the system
-// certificate pool.
+// WithSystemCerts return a grpc.DialOption for requiring TLS that is
```

Comment:
maybe "returns a grpc.DialOptions that uses the system-provided certificate authority chain to verify the required TLS connection" or somesuch?